### PR TITLE
Configure OCP cost as /month/node.

### PR DESCRIPTION
**Pull request by myersCody**
Example:
Endpoint: 
```
http://localhost:8000/api/cost-management/v1/reports/openshift/costs/?filter[resolution]=monthly&filter[time_scope_value]=-1&filter[time_scope_units]=month&group_by[cluster]=*&group_by[node]=*
```
Would Return: 
```
"data": [
        {
            "date": "2020-02",
            "clusters": [
                {
                    "cluster": "53E1AAD3-C9CB-48B8-B578-2E5E1D802E39",
                    "nodes": [
                        {
                            "node": "compute_1",
                            "values": [
                                {
                                    "node": "compute_1",
                                    "date": "2020-02",
                                    "cluster": "53E1AAD3-C9CB-48B8-B578-2E5E1D802E39",
                                    "cost": {
                                        "value": 84.467111,
                                        "units": "USD"
                                    },
                                    "infrastructure_cost": {
                                        "value": 0.0,
                                        "units": "USD"
                                    },
                                    "derived_cost": {
                                        "value": 84.467111,
                                        "units": "USD"
                                    },
                                    "markup_cost": {
                                        "value": 0.0,
                                        "units": "USD"
                                    },
                                    "cluster_alias": "test_F7E0C21F-542E-43D5-B084-1F57EE00DE50"
                                }
                            ]
                        },
                        {
                            "node": "compute_3",
                            "values": [
                                {
                                    "node": "compute_3",
                                    "date": "2020-02",
                                    "cluster": "53E1AAD3-C9CB-48B8-B578-2E5E1D802E39",
                                    "cost": {
                                        "value": 61.781666,
                                        "units": "USD"
                                    },
                                    "infrastructure_cost": {
                                        "value": 0.0,
                                        "units": "USD"
                                    },
                                    "derived_cost": {
                                        "value": 61.781666,
                                        "units": "USD"
                                    },
                                    "markup_cost": {
                                        "value": 0.0,
                                        "units": "USD"
                                    },
                                    "cluster_alias": "test_F7E0C21F-542E-43D5-B084-1F57EE00DE50"
                                }
                            ]
```

### CODE REVIEW

**Comment by adberglund** [comment]
Pulled this and ran some functional tests. We are very close, but something still isn't quite right. I loaded a cost model with a $1000/node/month cost for 1 of my 3 OpenShift clusters (`my-ocp-cluster-3`). The node/cluster breakdown is as follows 

```
koku=# select cluster_id, node from reporting_ocpusagelineitem_daily group by cluster_id, node order by cluster_id;
    cluster_id    |     node
------------------+---------------
 my-ocp-cluster-1 | compute_2
 my-ocp-cluster-1 | compute_3
 my-ocp-cluster-1 | compute_1
 my-ocp-cluster-1 | master
 my-ocp-cluster-2 | compute_2
 my-ocp-cluster-2 | compute_3
 my-ocp-cluster-2 | compute_1
 my-ocp-cluster-2 | master
 my-ocp-cluster-3 | node_movement
 my-ocp-cluster-3 | node_wait
 my-ocp-cluster-3 | node_action
```

So this means that I would expect $3000 and only on the my-ocp-cluster-3 cluster. We are correctly associating the cost with the my-ocp-cluster-3 cluster, but it is taking up other nodes too.

```
koku=# select cluster_id, node from reporting_ocpusagelineitem_daily_summary where monthly_cost is not null;
    cluster_id    |     node
------------------+---------------
 my-ocp-cluster-3 | compute_1
 my-ocp-cluster-3 | compute_2
 my-ocp-cluster-3 | compute_3
 my-ocp-cluster-3 | master
 my-ocp-cluster-3 | node_action
 my-ocp-cluster-3 | node_movement
 my-ocp-cluster-3 | node_wait
(7 rows)
```

File: koku/masu/database/ocp_report_db_accessor.py [hunk 1]
```diff
@@ -760,35 +771,46 @@ def populate_monthly_cost(self, node_cost, start_date, end_date):
...
             # Calculate monthly cost for every month
             for curr_month in rrule(freq=MONTHLY, until=end_date, dtstart=first_month):
                 first_curr_month, first_next_month = month_date_range_tuple(curr_month)
+                LOG.info('Populating Monthly node cost from %s to %s.',
+                         first_curr_month, first_next_month)
 
                 unique_nodes = OCPUsageLineItemDailySummary.objects.\
                     filter(usage_start__gte=first_curr_month,
                            usage_start__lt=first_next_month,
                            node__isnull=False
```
Comment (adberglund) [inline koku/masu/database/ocp_report_db_accessor.py]: I think we need to add `cluster_id=cluster_id` to this filter.

Comment (myersCody) [reply koku/masu/database/ocp_report_db_accessor.py]: ahhh. That is a great catch! 👍 I will make that change now.

Comment (myersCody) [reply koku/masu/database/ocp_report_db_accessor.py]: https://github.com/project-koku/koku/pull/1685/files#diff-5132f3be415e3032124d987a0aff4939R786

**Pull request merged**